packages/itwinui-css/backstop/tests/table.html

```diff
@@ -4616,22 +4616,54 @@ <h2>Empty state</h2>
         }
       };
 
+      const updateRowSelectionCount = (table) => {
+        const checkboxes = [...table.querySelectorAll('.iui-table-body .iui-table-row .iui-checkbox:not(:disabled)')];
+        const numBoxesChecked = checkboxes.filter((checkbox) => checkbox.checked).length;
+        const leftPaginator = table.querySelector('.iui-table-paginator .iui-left');
+        const paginatorCheckboxCount = leftPaginator.querySelector('.iui-table-paginator-selection-count');
+        if (!numBoxesChecked) {
+          if (paginatorCheckboxCount) {
+            leftPaginator.removeChild(paginatorCheckboxCount);
+          }
+          return;
+        }
+        const paginatorCheckboxCountText = numBoxesChecked === 1 ? '1 row selected' : `${numBoxesChecked} rows selected`;
+        if (!paginatorCheckboxCount) {
+          const newPaginatorCheckboxCount = document.createElement('span');
+          newPaginatorCheckboxCount.classList.add('iui-table-paginator-selection-count');
+          newPaginatorCheckboxCount.textContent = paginatorCheckboxCountText;
+          leftPaginator.appendChild(newPaginatorCheckboxCount);
+        }
+        else {
+          paginatorCheckboxCount.textContent = paginatorCheckboxCountText;
+        }
+      };
```

Comment:
Very nitpicky, but I'd put an empty line after this function.

Reply:
new line added. I forgot there's no stylelint for the demo pages